src/WebJobs.Extensions.DurableTask/ActorScheduler/SchedulerState.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using DurableTask.Core;
+using DurableTask.Core.Serializing;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// The persisted state of an actor scheduler, as handed forward between ContinueAsNew instances.
+    /// </summary>
+    [JsonConverter(typeof(CustomJsonConverter<SchedulerState>))]
+    internal class SchedulerState : CustomJsonConverter<SchedulerState>.IConversion
+    {
+        /// <summary>
+        /// Whether this actor exists or not.
+        /// </summary>
+        public bool ActorExists { get; set; }
+
+        /// <summary>
+        /// The serialized actor state. This can be stale while CurrentStateView != null.
+        /// </summary>
+        public string ActorState { get; set; }
```

Comment:
If these properties are intended to be serialized, let's explicitly mark them with `[JsonProperty]`. I would prefer it if we can set the `PropertyName` as well using a camelCase naming convention, which we typically use for all JSON payloads.

Reply:
I do see that you're using a custom JSON converter. I'll admit that I don't have much experience with those so I don't know whether `[JsonProperty]` is meant to be useful when custom converters are at play.

Reply:
Good catch. [JsonProperty] has no effect here because I am using a custom JsonConverter now. I will remove it.

Reply:
The custom JSON serialization has now been removed, and we are in fact using [JsonProperty] attributes.